client/src/components/App.css

```diff
@@ -6,47 +6,45 @@
   font-style: normal;
 }
 
+.big_grey_container {
```

Comment:
Is using underscores causing any issues with CSS modules or JS?

I asked because I saw a few suggestions to use camelCase to avoid issues.